src/object.d

```diff
@@ -3323,12 +3323,6 @@ unittest
         return _dup!(const(T), T)(a);
 }
 
-/// ditto
-@property T[] dup(T:void)(scope const(T)[] a) @trusted
-{
-    import core.internal.traits : Unconst;
-    return _dup!(const(T), Unconst!T)(a);
```

Comment:
@schveiguy interesting. Could you please add a "negative" unittest with the code above? (e.g. `static assert(!is(typeof(@safe { ... big lambda ... })));`